SUMMARY.md

```diff
@@ -51,6 +51,7 @@
 * [Application metrics](./content/chapter-03/ApplicationMetrics.md)
   * [Linux System Metrics](./content/chapter-03/LinuxSystemMetrics.md)
   * [Windows System Metrics](./content/chapter-03/WindowsSystemMetrics.md)
+* [Lab: common tools for system metrics](./content/chapter-03/SystemMetricsLab.md)
```

Comment:
[Lab: Tools for Measuring Resource Utilization](./content/chapter-03/SystemMetricsLab.md)

Reply:
Done